### ISSUE TITLE
Issue with required openssl version

### ISSUE THREAD

**Issue by assew** [philippe44/AirConnect#113]
Hi Philippe, I have tried installing your package on an older Raspi which has openssl 1.1.0f (25 May 2017) installed. I was not possible to finalise the install. I am not asking for specific support at this stage, but would like to know from a general perspective if - and why - having the older (pre-Heartbleed) openssl is really an imperative for getting your tool up and running? If so, can I ask which particular functionality in the old libssl versions is so important? Thank you for your feedback.

BTW, in addition, I am unsure which AirUpnp version I should use: 

$ sudo cat /proc/cpuinfo
processor       : 0
model name      : ARMv6-compatible processor rev 7 (v6l)
BogoMIPS        : 697.95
Features        : half thumb fastmult vfp edsp java tls
CPU implementer : 0x41
CPU architecture: 7
CPU variant     : 0x0
CPU part        : 0xb76
CPU revision    : 7

Hardware        : BCM2835
Revision        : 0002
Serial          : 000000009b9069e2

I tried with both airupnp-arm and airupnp-arm5 binaries, but got "Illegal Instruction" errors for both.

Thanks for your advice.

**Comment by philippe44** [comment]
You're making a good point for clarification, but I think that the version I'm pointing people to is the 1.0.1t, which is way pass the 1.0.1g which fixed heartbleed. The API compatibility is 1.0.0, hence the name and what I need, but the versio itself is safe (if you look at the build dat, 1.0.1t is 03-2018). 
Moving to a newer API is commplcated for a few reasons
1- I need to see what's different
2- I need to re-compile all my applications
3- A lot of people still today have 1.0.0 and I would have more complaints if I make 1.1.0 the default
So, yes I will surely change, but I want to have a reasonable guess that 1.1.0 is the majority of installled based

**Comment by assew** [comment]
Hi Philippe, thank you for your feedback. In the meantime, I got the application starting on another Linux/Debian box (using the x86-64 binary). It starts, but that's all. The first time I started, two upnp Players were detected, but no sound were played and now, this does not work any longer. I suspect there is a network issue. Switched of the firewall following the description here: https://wiki.fogproject.org/wiki/index.php/Disable_&_Verify_Firewall, but this had no effect. I suspect that either there is more switch-off-work to be done with respect to the udp ports universe or - as mentioned in several posts - I am "binding to the wrong interface". Is this e.g. using eth0 or eth1? However, I do not understand how I could instruct the application to use another interface. Could you help me out with that bit of knowledge and show a specific command line with the required parameters to effect the re-binding? Thank you in advance.

**Comment by philippe44** [comment]
the -h flag tells you the command line options.

**Comment by assew** [comment]
In the meantime, I learned that eth0 ... is outdated in Linux these days as it has been replaced by new interface names. The thing I was asking about was probably the "ip route list" command which showed me I had two routes on my box going nowhere. After those were removed, I could make a working Airplay connection at least once. My current tendency is to blame my rendering hardware and poor WLAN performance. Will continue trying to figure it out and eventually come back.
